src/styles/_variables.scss

```diff
@@ -20,6 +20,8 @@ $color-bg-dark-1: #001521;
 $color-bg-dark-2: #031a26;
 $color-bg-dark-3: #061c28;
 $color-bg-dark-4: #0e2330;
+$color-bg-dark-5: #2a3b46;
```

Comment:
This is going to cause conflicts with #286, unfortunately. One of us is going to have to sort out these colours and their order. We should come up with an official shared colour palette

Reply:
That PR has now been merged, so you're gonna have to rename some variables, sorry! btw are the shades still all in order from darkest to lightest? If not we should do that or else maybe come up with a new naming schema.

Reply:
I've not tried to keep them in any kind of order, so maybe not sorry. Honestly that seems like it would require additional refactoring every time we add new colours in between?

Reply:
I've found the order pretty useful personally (especially when trying to tweak the colours myself), but I agree that it might be a lot of work to refactor when adding new colours between. We might want to rename them to something more flexible (e.g. `$color-bg-dark-05, $color-bg-dark-10, $color-bg-dark-15`, or `$color-bg-very-dark, $color-bg-darker, $color-bg-darkest`, or whatever).

You'll need to rename the new ones in this case to resolve the merge conflict, but I'm happy to park the reordering for now, consider it tech debt, and revisit it later when we revisit the colour schema.

Reply:
Ok I'll handle the merge conflict and revisit the approach to colours later, since that seems to be an app-wide problem.